data/strings/strings.txt

```diff
@@ -5259,6 +5259,11 @@
     tags = android,ios
     en = [matrix]
 
+  [mastodon]
+    comment = Text in menu
+    tags = android
+    en = Mastodon
```

Comment:
Why do you think it should be in translatable strings? Maybe just hard-code it for simplicity now?

Reply:
can do, but adding a string is how the rest are done. should i hardcode the others too?
<img width="285" alt="Screenshot 2022-11-22 at 20 20 38" src="https://user-images.githubusercontent.com/26939824/203413383-99e17ecc-c21c-4a28-bbc9-64ad88ab6d0b.png">

Reply:
That's unexpected, for the Chinese :) Ok, let's leave it as is for now.